model-optimizer/extensions/front/tf/ObjectDetectionAPI.py

```diff
@@ -757,7 +757,7 @@ def transform_graph(self, graph: Graph, replacement_descriptions: dict):
                 # replace sub-graph between start and end nodes (including them) with new_preprocessing_ops nodes
                 end_node.out_port(0).get_connection().set_source(new_preprocessing_ops[-1].out_port(0))
                 start_node.in_port(0).get_connection().set_destination(
-                    new_preprocessing_ops[0].in_port(new_preprocessing_ops[0].is_in_port_connected(0)))
+                    new_preprocessing_ops[0].in_port(0 if not new_preprocessing_ops[0].is_in_port_connected(0) else 1))
```

Comment:
@lazarevevgeny can you, please, comment, is this valid fix? Before this change the transformation creates ipput port with id = `False`

Reply:
I would recommend to change the code the following way:
```python
new_preprocessing_ops[0].in_port(int(new_preprocessing_ops[0].is_in_port_connected(0))))
```

I have added conversion of the boolean value to int explicitly.

Reply:
Ok, I will update the transformation this way

Reply:
Done.